.travis.yml

```diff
@@ -57,6 +57,7 @@ env:
     - JHI_APP=jdl-default JHI_ENTITY=jdl JHI_PROFILE=prod JHI_PROTRACTOR=1
     - JHI_APP=ngx-default JHI_PROFILE=prod JHI_PROTRACTOR=1 JHI_ENTITY=sql JHI_SONAR=1
     - JHI_APP=ngx-psql-es-noi18n JHI_PROFILE=prod JHI_PROTRACTOR=1 JHI_ENTITY=sqlfull
+    - JHI_APP=ngx-psql-es-noi18n-mapsid JHI_PROFILE=prod JHI_PROTRACTOR=1 JHI_ENTITY=sqlfull
```

Comment:
Could we add these new entities to `ngx-psql-es-noi18n` builds instead of creating a new one ?

Reply:
Yes! I will use existing one.

Reply:
I have merged and kept suffix name `-mapsid` for clarification. I do not think that this will make any impact. Am I right?

Reply:
In fact, there are impacts, for Azure daily builds:
- https://github.com/hipster-labs/jhipster-daily-builds/blob/master/azure-pipelines-official.yml#L50
- https://github.com/hipster-labs/jhipster-daily-builds/blob/master/azure-pipelines-elasticsearch.yml#L45

But it can be changed there.